STYLEGUIDE.md

```diff
@@ -380,23 +380,23 @@ Use lodash and jQuery libraries in `/apps`.
 
 ## CSS
 
-Default: https://github.com/thoughtbot/guides/tree/master/style/sass
-
-Fallback: https://css-tricks.com/sass-style-guide
-
-Fallback: http://css-tricks.com/css-style-guides
+On the frontend, some lint and style rules are enforced by [Stylelint](https://stylelint.io/). You can check your changes locally by running `yarn lint:scss` from the `apps` directory. Some rules have been disabled for now, but we are moving towards enabling all of the rules that are part of the Stylelint standard configuration.
 
+Some key points:
+- Use SCSS modules over CSS-in-JS. The module file takes the name "my-component.module.scss" and lives in the same directory as the component.
+- Use kebab-case (not camelCase nor snake_case) for separating words in IDs, classes, mixins and filenames. 
 - Avoid inline styles in markup.
 - Use names that are as short as possible but as long as necessary.
 - Use SCSS helpers for vendor prefixing.
 - Extract colors into named, re-used variables.
 - Extract magic numbers into named variables.
 - Use ID selectors only when there is, and only ever will be, one item on a page.
 - Nest selectors a maximum of three levels deep.
-- Use dashes instead of underscores, camel casing, etc for separating words in IDs, classes, mixins and filenames.
```

Comment:
@bencodeorg I was modifying this to just say "kebab-case" ... but clearly we don't follow this guideline consistently in our codebase